packages/icons/src/library/line-dashed.js

```diff
@@ -4,7 +4,7 @@
 import { Path, SVG } from '@wordpress/primitives';
 
 const lineDashed = (
-	<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none">
+	<SVG xmlns="http://www.w3.org/2000/svg" fill="none">
```

Comment:
Should we add `viewBox="0 0 24 24"` to the `icons` that don't have it? --cc @jameskoster

Reply:
Yes I believe so 👍

Reply:
Sure thing, will check the others as well.

Reply:
Do they all need a `fill="none"` as well, or is that conditionally based on the icons design?

Reply:
Also found the format-ltr & format-rtl have non standard sizes of `viewBox="-2 -2 24 24"`, hesitant to change those without some confirmation

Reply:
Same for `insertAfter`, `insertBefore`,`keyboardClose`,`keyboardReturn`,`plusCircle`,`tableColumnAfter`, `tableColumnBefore`,`tableColumnDelete`, `tableRowAfter`, `tableRowBefore`, `tableRowDelete`, `update`, `warning`,  & finally `wordpress`.

Sorry thought there were less when I started, might as well be comprehensive once I got halfway through lol.

---

Along the lines of standardizing, I went ahead and sorted props the same way for each with `xmlns` followed by `viewBox` and finally `fill`

Reply:
For reference, standardizing order of props makes it easier to use Find & Replace to look for discrepencies.

Reply:
Thanks for the changes! I don't think changing the order was needed in this PR, but there is no harm in it. Initially I meant add it only in the updated icons.

You should rebase with `trunk` because there are conflicts from `icons` that have been removed now and it should be good to land.